homeassistant/components/shelly/sensor.py

```diff
@@ -144,6 +145,15 @@
         device_class=sensor.DEVICE_CLASS_ILLUMINANCE,
     ),
     ("sensor", "tilt"): BlockAttributeDescription(name="tilt", unit=DEGREE),
+    ("relay", "totalWorkTime"): BlockAttributeDescription(
+        name="Lamp life",
+        unit=PERCENTAGE,
+        icon="mdi:gauge",
```

Comment:
maybe this icon will be a better fit? 
https://materialdesignicons.com/icon/progress-wrench

Reply:
```suggestion
        icon="mdi:progress-wrench",
```